metatron-app/metatron-discovery
# Fixed screen error

### Description
1. Black theme Exploration title color is not white
2. When scrollbars appear on the dashboard filter layer, overlap the bottom text and lines
3. In engine monitoring, key and value overlap.
4. Button status is hard to see on dashboard status screen.
5. There is no bottom margin in Administration> Workspaces.
6. The content is displayed outside of the Administration> Users> Members pop-up.
7. The pop-up menu in the Filter panel is too wide to navigate.

**Related Issue** : 

### How Has This Been Tested?
1. Check if the explosion title is visible in the black theme.
2. If scrollbars appear on the dashboard filter layer, make sure the bottom text is visible without overlapping.
3. Under Engine Monitoring, verify that the content is displayed without overlapping.
4. Check if the button text is visible on the dashboard status error screen.
5. Verify that there is an Administration> Workspaces bottom margin.
6. Verify that content is displayed within the Administration> Users> Members pop-up.
7. Make sure the pop-up menu in the Filter panel is narrow.

#### Need additional checks?

### Types of changes


### Checklist:


### Additional Context
